infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -300,19 +299,17 @@ class GeometryService @Autowired constructor(
     }
 
     @Scheduled(cron = "\${geoviite.rail-network-export.schedule}")
-    @Scheduled(initialDelay = 1000 * 300, fixedDelay = Long.MAX_VALUE)
+    @Scheduled(initialDelay = 1000 * 3, fixedDelay = Long.MAX_VALUE)
```

Comment:
Jos luen tätä oikein niin tässä on nyt kolmen sekunnin delay aiemmin 5min sijasta. Onko tämä tarkoituksella (tai jos delayt on tuota luokkaa, niin onko niille ylipäätään tarvetta?)

Reply:
Ei ole tarkoituksella, hyvä bongaus. Testailin näitä ja unohtui :see_no_evil: